Makefile

```diff
@@ -53,12 +54,12 @@ test: $(NODE_MODULES)  ## Runs the tests
 check: $(NODE_MODULES) test lint  ## Runs all checks required before committing
 
 clean:  ## Cleans up everything
-	rm -rf node_modules .*-updated .*-bin out static/dist static/vs
+	rm -rf node_modules .*-updated .*-bin out dist
```

Comment:
Ideally I'd prefer `dist` to live under `out` to not litter the top directory with generated things, but there may be a reason, and I'm not against it :)

Reply:
cheers!